packages/rum-core/src/common/patching/fetch-patch.js

```diff
@@ -105,3 +106,7 @@ export function patchFetch(callback) {
     })
   }
 }
+
+export function getNativeFetch() {
```

Comment:
As we commented in the call, this was the mechanism to avoid creating transactions when using fetch the agent.

Despite that, a different strategy to avoid creating needless transactions has been created.